libs/environment/src/components/node-guard/NodeGuard.graphql

```diff
@@ -0,0 +1,13 @@
+query NodeGuard {
+  statistics {
+    blockHeight
+  }
```

Comment:
you could use lastBlockHeight here, being not proxied to the core node, it should be faster